device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -478,7 +478,7 @@ public void close() throws IOException
             if (this.transportClient.getTransportClientState() == TransportClient.TransportClientState.OPENED)
             {
                 // Codes_SRS_DEVICECLIENT_12_006: [If the client has been initialized to use TransportClient and the TransportClient is already opened the function shall throw an IOException.]
-                throw new IOException("Calling closeNow() when using TransportClient is not supported. Use TransportClient.closeNow() instead.");
+                throw new UnsupportedOperationException("Calling closeNow() when using TransportClient is not supported. Use TransportClient.closeNow() instead.");
```

Comment:
Should this be documented up by 466?

Reply:
wouldn't we want to add the exception to the throws keyword?

Reply:
```UnsupportedOperationException``` is a runtime exception, and we typically don't declare those in the method declaration since Java doesn't require it. This is basically the same as how we have APIs that throw IllegalArgumentException but that don't list it in the declared thrown exceptions list